src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/RejectDuplicatesTask.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.blobrouter.tasks;
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.slf4j.Logger;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.blobrouter.services.storage.DuplicateFileHandler;
+
+import static org.slf4j.LoggerFactory.getLogger;
+import static uk.gov.hmcts.reform.blobrouter.util.TimeZones.EUROPE_LONDON;
+
+@Component
+@ConditionalOnProperty(value = "scheduling.task.reject-duplicates.enabled")
+public class RejectDuplicatesTask {
```

Comment:
There should be a unit test for RejectDuplicatesTask

Reply:
done